salt/modules/gpg.py

```diff
@@ -789,6 +828,12 @@ def export_key(keyids=None, secret=False, user=None, gnupghome=None):
     gnupghome
         Specify the location where GPG keyring and related files are stored.
 
+    use_passphrase
+        Whether to use a passphrase with the signing key. Passphrase is received
+        from Pillar.
+
+        .. versionadded:: 3003
+
```

Comment:
```suggestion
    use_passphrase

        .. versionadded:: 3003
        
        Whether to use a passphrase with the signing key. Passphrase is received
        from Pillar.

```

Reply:
The initial problem were spaces in the empty line before `.. versionadded`, so it fails with trailing spaces errors. Fixed now.